### PR TITLE
Add NVIDIA model samples

### PR DESCRIPTION
**Pull request by skasturi**
# Description

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: correct_test_task_type
```
Comment (skasturi) [inline cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:3]: @HrishikeshGeedMS Could you please change these?

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+description: correct task type testing
+
+
+inputs:
+  compute: do-not-delete-nvidia-testing
```
Comment (skasturi) [inline cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:8]: @vikasagrawal-ms Rename?

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  model_path: 
+    type: triton_model
+    path: azureml://registries/nvidia-ai/models/Nemotron-3-8B-Base-4k/versions/1 
+  train_dataset_path: 
+    type: uri_folder
+    path: azureml:train_tg_nv:1
```
Comment (skasturi) [inline cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:14]: @vikasagrawal-ms  Do we have this as part of the notebook?

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  valid_dataset_path: 
+    type: uri_folder
+    path: azureml:val_tg_nv:1
+  ptuned_model_name: ptuned_model
+  input_column_name: text
+  target_column_name: summary
```
Comment (skasturi) [inline cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:20]: @vikasagrawal-ms Do we support summarization in addition to text-gen?

File: cli/foundation-models/nvidia-nemo/nemo_ptuning_summarization_with_textgen.ps1 [hunk 5]
```diff
@@ -0,0 +1,7 @@
+$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest -Uri https://aka.ms/installazurecliwindows -OutFile .\AzureCLI.msi; Start-Process msiexec.exe -Wait -ArgumentList '/I AzureCLI.msi /quiet'; Remove-Item .\AzureCLI.msi
+
+az extension add -n ml
+
+az login
+
+az ml job create -g <RESOURCE_GROUP> -w <WORKSPACE> --subscription <SUBSCRIPTION> -f nemo_pipeline_job.yml
```
Comment (skasturi) [inline cli/foundation-models/nvidia-nemo/nemo_ptuning_summarization_with_textgen.ps1:7]: @vikasagrawal-ms Could we please get this working out of the box?

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml (hunk 1 above)
Comment (HrishikeshGeedMS) [reply cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:3]: Sure changing it

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml (hunk 4 above)
Comment (vikasagrawal-ms) [reply cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:20]: summarization is modelled as text-gen

File: cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml (hunk 3 above)
Comment (vikasagrawal-ms) [reply cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml:14]: in notebook we are using samsum dataset

File: cli/foundation-models/nvidia-nemo/nemo_ptuning_summarization_with_textgen.ps1 (hunk 5 above)
Comment (vikasagrawal-ms) [reply cli/foundation-models/nvidia-nemo/nemo_ptuning_summarization_with_textgen.ps1:7]: Do we want to add, creation of compute as well in this.

**Review by Addi-11 — APPROVED** [review]

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**